processing/src/main/java/io/druid/query/metadata/SegmentMetadataQueryQueryToolChest.java

```diff
@@ -77,6 +79,36 @@ public SegmentMetadataQueryQueryToolChest(
   {
     return new ResultMergeQueryRunner<SegmentAnalysis>(runner)
     {
+      private Function<SegmentAnalysis, SegmentAnalysis> transformFn = new Function<SegmentAnalysis, SegmentAnalysis>()
+      {
+        @Nullable
+        @Override
+        public SegmentAnalysis apply(@Nullable SegmentAnalysis analysis)
```

Comment:
@himanshug Hm, yes, it was added by IDE since it was part of the Function interface definition. I've removed the @Nullables.